realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncedRealmTests.kt

```diff
@@ -554,8 +578,10 @@ class SyncedRealmTests {
                     columnDecimal128 = expectedDecimal128
                     columnObjectId = expectedObjectId
                     columnUUID = expectedUUID
+                    columnMixed = expectedMixed
```

Comment:
Just for completion I think we should just iterate all the types and see if they sync properly for a single field. We could just add a loop with a `realm1sesseion.upload...()` followed by `realm2session.download...() at the end of this test.